src/geo/cartodb-layer-common.js

```diff
@@ -97,6 +97,7 @@ CartoDBLayerCommon.prototype = {
     return this;
   },
 
+  // TODO: This can be removed
```

Comment:
So, can the method be removed? Otherwise I'd say the comment should be removed, or clarify what's pending for it to be removed or such.